internal/sdkv2provider/resource_cloudflare_workers_kv.go

```diff
@@ -101,14 +101,13 @@ func resourceCloudflareWorkersKVDelete(ctx context.Context, d *schema.ResourceDa
 }
 
 func resourceCloudflareWorkersKVImport(ctx context.Context, d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
-	namespaceID, key, err := parseId(d.Id())
+	accountID, id, err := parseId(d.Id())
```

Comment:
`parseId` no longer seems suitable here given the method only splits into 2 sections and the import format needs 3. i.e. `<account_id>/<namespace_id>/<key_name>`

Reply:
> `parseId` no longer seems suitable here given the method only splits into 2 sections and the import format needs 3. i.e. `<account_id>/<namespace_id>/<key_name>`

@jacobbednarz 

Sorry. Looking back it's not my proudest work. I rushed a bit.

I've done some more thorough testing and needed to add a couple of lines so the future `plan`s don't try and recreate the resource.

How is that?